src/mlpack/methods/ann/layer/multiply_merge.hpp

```diff
@@ -135,6 +135,12 @@ class MultiplyMerge
   //! Modify the parameters.
   OutputDataType& Parameters() { return weights; }
 
+  //! Get the value of model parameter.
+  bool const& Model() const { return model; }
+
+  //! Get the value of run parameter.
+  bool const& Run() const { return run; }
+
```

Comment:
Looks like ```run``` is not something that might benefit with a getter function.
On the other hand, having a getter coupled with a modifier might make sense if the user wants to manually shift between ```forward``` and ```backward```. Any thoughts on this @zoq?

Reply:
I am going ahead and adding a setter for this now. If required I can remove it in a later commit.